src/main/java/edu/ksu/canvas/attendance/controller/AttendanceHelpController.java

```diff
@@ -0,0 +1,35 @@
+package edu.ksu.canvas.attendance.controller;
+
+
+import edu.ksu.lti.launch.exception.NoLtiSessionException;
+import org.apache.commons.validator.routines.LongValidator;
+import org.apache.log4j.Logger;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@Scope("session")
+@RequestMapping("/help")
+public class AttendanceHelpController extends AttendanceBaseController{
+
+    private static final Logger LOG = Logger.getLogger(AttendanceHelpController.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)